core/templates/core/user-post-list.html

```diff
@@ -0,0 +1,83 @@
+{% extends 'users/profile.html' %}
+{% load static %}
+
+{% block extra_css %}
```

Comment:
I think I would throw it into users/profile.html.  So we don't need to add it on those subpages. 
Your call.